src/widget/wlibrary.cpp

```diff
@@ -39,7 +39,12 @@ void WLibrary::switchToView(const QString& name) {
         if (widget != NULL && currentWidget() != widget) {
             //qDebug() << "WLibrary::setCurrentWidget" << name;
             setCurrentWidget(widget);
-            dynamic_cast<LibraryView*>(widget)->onShow();
+            LibraryView * lview = dynamic_cast<LibraryView*>(widget);
+              if (lview == NULL) {
```

Comment:
The dynamic cast should be done above the if statement above `if (widget !=..`
Please check also indent and brake around column ~80